Dockerfile.arm64

```diff
@@ -1,5 +1,4 @@
-FROM alexellis2/golang:1.9-arm64 as build
-#FROM golang:1.7.5
+FROM golang:1.9.2 as build
```

Comment:
Should work but can we verify on Scaleway?

Reply:
I think I have verified on scaleway. I can confirm once again.